test/cppsuite/test_harness/runtime_monitor.cxx

```diff
@@ -36,16 +36,6 @@
 
 namespace test_harness {
 /* Static methods implementation. */
-static void
```

Comment:
Do we still need that change now we have a stat_cursor in per thread context ?

Reply:
yes i think we do, since we need to call `stat_cursor` outside of this file. I believe local static functions can only be accessed within the file it is defined.

Reply:
I have changed it from static to just a normal file function. What do you think of this approach?

Reply:
I think it can be `static` as it does not use any class member.